src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -19,6 +19,17 @@ abstract class PipelineTargets(config: Config) {
       "writeOpsMetrics, lastOptimizedTS, Pipeline_SnapTS, primordialDateString").split(", ")
   )
 
+  val apiEventsTarget: PipelineTable = PipelineTable(
+    name = "api_events_meta",
+    _keys = Array("organization_id", "moduleId", "Overwatch_RunID"),
+    config = config,
+    partitionBy = Array("organization_id"),
+    incrementalColumns = Array("Pipeline_SnapTS"),
```

Comment:
what is the incremental going to be? What is the writeMethod? I'm guessing it's append only -- is there an incremental? think through this

Reply:
its append only